src/main/java/fruitshop/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,18 @@
+package fruitshop.strategy;
+
+import fruitshop.db.Storage;
+import fruitshop.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
The class `ReturnOperation` should be in the `strategy` package, which is correct. Ensure that other related classes are also placed in their respective packages according to the checklist.